src/LaunchDarkly.ClientSdk/PlatformSpecific/Connectivity.maui.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Maui.Networking;
+
+namespace LaunchDarkly.Sdk.Client.PlatformSpecific
+{
+    internal static partial class PlatformConnectivity
+    {
+        static PlatformConnectivity()
+        {
+            Connectivity.Current.ConnectivityChanged += (sender, args) => ConnectivityChanged?.Invoke(sender, EventArgs.Empty);
```

Comment:
For reviewers: Intentionally not forward args to simplify number of unique container classes.  The args are ignored today.